daft/viz/tabulate.py

```diff
@@ -0,0 +1,22 @@
+"""MONKEY PATCHED TABULATE."""
+
+import tabulate
+from tabulate import DataRow, Line, TableFormat
+
+# add a default which is like our rust comfytable
+tabulate._table_formats["default"] = TableFormat(
+    lineabove=Line("╭", "─", "┬", "╮"),
+    linebelowheader=Line("╞", "═", "╪", "╡"),
+    linebetweenrows=Line("├", "╌", "┼", "┤"),
+    linebelow=Line("╰", "─", "┴", "╯"),
+    headerrow=DataRow("│", "┆", "│"),
+    datarow=DataRow("│", "┆", "│"),
+    padding=1,
+    with_header_hide=None,
+)
```

Comment:
instead of having 2 different table implementations, why not push the options down in to rust? comfytable has many of the same configuration options.

Reply:
I considered this, but I saw comfy-table has few builtin formatting options whereas this gave us a bunch of formats like markdown out of the box. I can move this logic to comfy-table, but it would be a lot more work to get the same functionality.

Reply:
I feel a good argument for doing it in rust is (1) more control and (2) no additional customer dep. If we agree with the formatting options and arguments, then I can move the logic over!

Reply:
switched to rust. https://github.com/Eventual-Inc/Daft/pull/4100